JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -1,16 +1,122 @@
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func strawberryJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .strawberryJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func bananaJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .bananaJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func pineappleJuieButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .pineappleJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
     }
     
+    @IBAction func kiwiJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .kiwiJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func mangoJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .mangoJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func mankiJuiceButtonTapped(_ sender: UIButton) {
+        let orderedJuice: JuiceMenu = .mangoKiwiJuice
+        if JuiceMaker.sharedJuiceMaker.makeJuice(orderedJuice) {
+            updateStockLabel()
+            successAlert(name: orderedJuice.juiceName)
+        } else {
+            failAlert()
+        }
+    }
+    
+    @IBAction func barButtonTapped(_ sender: UIBarButtonItem) {
+        performSegue(withIdentifier: "toSecondVC", sender: self)
+    }
+    
+    func successAlert(name: String) {
+        let alert = UIAlertController(title: "\(name) 나왔습니다! 맛있게 드세요!",message: nil, preferredStyle: UIAlertController.Style.alert)
+        let cancel = UIAlertAction(title: "확인", style: .default)
+        alert.addAction(cancel)
+        present(alert, animated: true, completion: nil)
+    }
+    
+    func failAlert() {
+        let alert = UIAlertController(title: "재료가 모자라요. 재고를 수정할까요?", message: nil, preferredStyle: UIAlertController.Style.alert)
+        let ok = UIAlertAction(title: "예", style: .default, handler: { action in self.performSegue(withIdentifier: "toSecondVC", sender: nil) })
```

Comment:
```suggestion
        let ok = UIAlertAction(title: "예", style: .default, handler: { action in 
            self.performSegue(withIdentifier: "toSecondVC", sender: nil) 
        })
```

코드 한 줄이 너무 긴 느낌이 있는데 개행을 넣어도 좋을 것 같아요~

Reply:
네! 코드에 반영했습니다